src/scanspec/specs.py

```diff
@@ -45,6 +42,12 @@
 #: Can be used as a special key to indicate how long each point should be
 DURATION = "DURATION"
 
+#: A type variable for an `axis_` that can be specified for a scan
+Axis = TypeVar("Axis", covariant=True)
```

Comment:
Why re-implement rather than import?

Reply:
My mistake, I thought it was best practice, but having researched a bit more I agree that importing `TypeVars` is the way to go. Case in point: When I removed these it broke because `Axis` in `core.py` wasn't covariant. I have now fixed and imported both variables.